docs/contributing/contributing-guide.md

```diff
@@ -0,0 +1,106 @@
+# Contributing guide
+
+All contributions, bug reports, bug fixes, documentation improvements, and enhancements are welcome.
+
+All contributors and maintainers to this project should abide by the [Contributor Code of Conduct](/docs/contributing/code-of-conduct.md).
+
+Learn more about the contributors' roles in the [Roles](/docs/contributing/contributor-roles.md) page.
+
+This document describes how to contribute to the Great Expectations Airflow Provider, covering:
+
+- Overview of how to contribute
+- How to set up the local development environment
+- Running tests
+- Authoring the documentation
+
+## Overview of how to contribute
+
+To contribute to the Great Expectations Airflow Provider project:
+
+1. Create a [GitHub Issue](https://github.com/astronomer/airflow-provider-great-expectations/issues) describing a bug, enhancement, or feature request.
+2. Open a branch off of the `main` branch and create a Pull Request into the `main` branch from your feature branch.
```

Comment:
contributors outside of astronomer will need to fork the repo and create their branch within their fork.